src/shared/styles/globals.css

```
@@ -4,7 +4,10 @@
 
 html,
 body {
-  font-family: NanumSquareNeo, Pretendard, sans-serif;
+  font-family: Pretendard, NanumSquareNeo, sans-serif;
+  font-family: Pretendard, sans-serif;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-family is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)